packages/design-tokens/stories/Shadows.stories.tsx

```diff
@@ -14,7 +14,7 @@ const ShadowSwatch: React.FC<ShadowSwatchProps> = ({
   className = '',
 }) => (
   <div
-    className={`grid h-[100px] place-content-center rounded bg-default text-center ${className}`}
+    className={`grid h-24 place-content-center rounded bg-default text-center ${className}`}
```

Comment:
Even though we are allowing arbitrary values we should try to use the spacing and size system provided by tailwind as much as possible to reduce the amount of CSS that is generated